tests/helpers.py

```diff
@@ -1,16 +1,24 @@
 from brownie import chain
 
+H = 3600
 DAY = 86400
 WEEK = 7 * DAY
+MAXTIME = 4 * 365 * DAY
+TOL = 120 / WEEK
+
 
 def approx(a, b, precision=1e-10):
     if a == b == 0:
         return True
+    print(2 * abs(a - b) / (a + b))
+    print(precision)
     return 2 * abs(a - b) / (a + b) <= precision
```

Comment:
Notice that this method of computing differences will crash if both the actual and the expected are both zero. ( `a+b` will equal zero, which will cause a divide by zero error.)

This math does seem to work for small differences, > 3 decimals, but it gets weird for low decimal precision. The difference between 50 and 100 comes out to 0.66666, which probably isn't what you would expect from such a large difference.

Reply:
good feedback thanks. have changed the function to: 
- always return false if expected / received numbers have different sign
- instead of a / b have renamed properties as `expected`and `actual`. This way the denominate using the `exptected` property and not the average between expected and actual. so 50 (expected) and 100(actual) will yield `2` as in 200%. Which is more correct IMHO